src/components/product-details/product-details.hbs

```diff
@@ -1,129 +1,129 @@
 <div class="product-details" data-component="product-details">
-  <div class="product-description">
-    <h1 class="product-description__title">{{product_name}}</h1>
-    <span class="product-description__price" aria-label="product price">
-      <span class="product-description__price-now">
-        <span class="price__label" aria-hidden="true">Price</span>
-        <span class="price__dollar" aria-hidden="true">${{price_sale}}</span>
-        <span class="element-invisible">current price ${{price_sale}}</span>
-      </span>
+	<div class="product-description">
+		<h1 class="product-description__title">{{product_name}}</h1>
+		<span class="product-description__price" data-js="gallery-target-sibling" aria-label="product price">
+			<span class="product-description__price-now">
+				<span class="price__label" aria-hidden="true">Price</span>
+				<span class="price__dollar" aria-hidden="true">${{price_sale}}</span>
+				<span class="element-invisible">current price ${{price_sale}}</span>
+			</span>
+			<span class="product-description__price-was">
+				<span class="price__label" aria-hidden="true">was</span>
+				<span class="price__dollar" aria-hidden="true">${{price_actual}}</span>
+				<span class="element-invisible">was ${{price_actual}}</span>
+			</span>
+			<span class="product-description__price-save">
+				<span class="price__label" aria-hidden="true">save</span>
+				<span class="price__dollar" aria-hidden="true">${{price_savings}}.00</span>
+				<span class="element-invisible">save ${{price_savings}}</span>
+			</span>
+		</span>
+		<hr class="divider divider--medium" aria-hidden="true" />
+		<p class="product-description__summary">
+			{{short_desc}}
+		</p>
+		<a href="#product-description" class="product-description__read-more">Full product description</a>
 
-      <span class="product-description__price-was">
-        <span class="price__label" aria-hidden="true">was</span>
-        <span class="price__dollar" aria-hidden="true">${{price_actual}}</span>
-        <span class="element-invisible">was ${{price_actual}}</span>
-      </span>
-      <span class="product-description__price-save">
-        <span class="price__label" aria-hidden="true">save</span>
-        <span class="price__dollar" aria-hidden="true">${{price_savings}}</span>
-        <span class="element-invisible">save ${{price_savings}}</span>
-      </span>
-    </span>
-    <p class="product-description__summary">
-      {{short_desc}}
-      <br />
-      <a href="#product-description" class="product-description__read-more">Full product description</a>
-    </p>
+		<form class="product-form" data-component="product-form">
+			<hr class="divider" aria-hidden="true" />
```

Comment:
@aevanson why did you aria hide this hr?

Reply:
Well, they get read out by the screen reader like "HORIZONTAL SEPARATOR". I didn't think it was necessary to hear that, so I've hid them from the screen reader like decorative graphics